markdown/3.x/en/ff-filter-cloud.md

```diff
@@ -0,0 +1,99 @@
+## Using the Filter Cloud
+You can use the Filter Cloud by adding one line to your HTML code.
+```html
+<ff-filter-cloud></ff-filter-cloud>
+```
+Doing so will cause the Filter Cloud to use default HTML template, that is `<span data-template="filter">{{element.name}}</span>`, for all filters. Filter Cloud is meant to be used together with ASN, so it will work only when `ff-asn` is present on the page.
+
+
+### Rendered HTML
+When setup like in the example above, the rendered HTML could look like this:
+```html
+<ff-filter-cloud>
+    <span data-template="filter">Element 1</span>
+    <span data-template="filter">Element 2</span>
+    <span data-template="filter">Element 3</span>
+    <span data-template="filter">Element 4</span>
+</ff-filter-cloud>
+```
+
+## Adding a filter template
+To customize filter element appearance, you can add an HTML element annotated with `[data-template-filter]`. You can access the filter group's name through `{{group.name}}`:
+```html
+<ff-filter-cloud>
+    <span data-template="filter">{{group.name}}: {{element.name}}</span>
+</ff-filter-cloud>
+```
+
+### Rendered HTML
```

Comment:
When using `### Rendered HTML`, you also need `### Setup`. The next setup is currently under the previous _Rendered_ section. See [specifications](https://github.com/FACT-Finder-Web-Components/feature-lab/blob/ff-filter-cloud/specs/ff-asn-selected-filter-cloud/ff-asn-selected-filter-cloud.md#specifying-templates).

Reply:
Fixed.